bin/unwanted_biotypes.txt

```diff
@@ -0,0 +1,55 @@
+IG_C_gene
```

Comment:
is this specific to human and mouse?

Reply:
Yes. I can poke around and check if other iGenomes species have as much biotypes floating around in the GTF file. It makes annotation very difficult with them present